File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int sSum = 0, dSum = 0;
        
        for (int num : nums) {
            if (num <= 9) {
                sSum += num;
            } else {
                dSum += num;
            }
        }
        
        return (sSum > dSum || dSum > sSum);
    }
}

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution(object):
    def canAliceWin(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        single=0
        double=0
        for i in range(len(nums)):
            if(nums[i]<=9):
                single+=nums[i]
            else:
                double+=nums[i]
        if(double >single or single>double):
            return True
        return False